tests/testTriggers.go

```diff
@@ -5967,22 +5971,24 @@ func TriggerStorkApplicationBackup(contexts *[]*scheduler.Context, recordChan *c
 		log.Infof("Backup applications, present in namespaces - %v", backupNamespaces)
 		log.Infof("Start backup application")
 		for i, currbkNamespace := range backupNamespaces {
-			taskNamePrefix = taskNamePrefix + fmt.Sprintf("%d", i)
-			currBackupLocation, err := applicationbackup.CreateBackupLocation(taskNamePrefix+"-location", currbkNamespace, s3SecretName)
+			taskNamePrefix = taskNamePrefix + fmt.Sprintf("-%d", i)
+			backuplocationname := taskNamePrefix + "-location" + time.Now().Format("15h03m05s")
+			backupname := taskNamePrefix + time.Now().Format("15h03m05s")
+			currBackupLocation, err := applicationbackup.CreateBackupLocation(backuplocationname, currbkNamespace, s3SecretName)
 			if err != nil {
 				UpdateOutcome(event, fmt.Errorf("backup location creation failed with %v", err))
 				return
 			}
-			bkp, bkp_create_err := applicationbackup.CreateApplicationBackup(taskNamePrefix+"-backup", currbkNamespace, currBackupLocation)
+			_, bkp_create_err := applicationbackup.CreateApplicationBackup(backupname, currbkNamespace, currBackupLocation)
```

Comment:
Here you need to get the object created by the `CreateApplicationBackup` API and then use that object in the rest of the code.

example:

```
bkp, bkp_create_err := applicationbackup.CreateApplicationBackup(backupname, currbkNamespace, currBackupLocation)
````

and going forward use, bkp.Name and bkp.Namespace etc.

Reply:
Thanks Rohit, As discussed, I will make these changes with my pool resize PR.